packages/qdrant-loader/src/qdrant_loader/webhooks/server.py

```
@@ -239,12 +239,14 @@ async def readyz() -> dict[str, object]:
                 status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
                 detail=f"Worker task failed: {str(e)}",
             )
-    
+
     return {"status": "ready"}
 
 
 @app.get("/status", dependencies=[Depends(verify_cognito_token)])
-async def status_route(claims: dict[str, Any] = Depends(verify_cognito_token)) -> dict[str, object]:
+async def status_route(
+    claims: dict[str, Any] = Depends(verify_cognito_token),
```

Ruff (0.15.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)